test/vsphere/e2e.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit  # exits immediately on any unexpected error (does not bypass traps)
+set -o nounset  # will error if variables are used without first being defined
+set -o pipefail # any non-zero exit code in a piped command causes the pipeline to fail with that code
+
+ARTIFACTS=$PWD/artifacts
+CERTS=$PWD/test/vsphere/openvpn/certs
+
+# Binary
+GO_VERSION=${GO_VERSION:-1.12}
+SOPS_VERSION=${SOPS_VERSION:-3.5.0}
+
+install_platformcli() {
+  docker run --rm -it -v "$PWD":"$PWD" -v /go:/go \
+    -w "$PWD" --entrypoint make -e GOPROXY=https://proxy.golang.org \
+    golang:"$GO_VERSION" pack build
+
+  sudo mv "$PWD"/.bin/platform-cli /usr/local/bin/platform-cli
+  sudo chmod +x /usr/local/bin/platform-cli
+  sudo apt-get update
+  sudo apt-get install -y genisoimage
+}
+
+install_kubectl() {
```

Comment:
I don't think this is required  - kubectl should be downloaded on-demand by platform-cli

Reply:
Yeah true I just put it there to run some tests when dependencies could not be installed.